backend/api/api/views.py

```diff
@@ -0,0 +1,36 @@
+from django.http import HttpResponse
+
+#importing functions from folder
+import backend.functions as functions
+from functions.register import register
+from functions.login import login
+
+current_user = ''
+
+def register_user(request):
+    result = 1
+    if request.method == 'POST':
+        username = request.POST['username']
+        password = request.POST['password']
+        result = register(username, password)
+    if result == 0:
+        pass #should redirect to new page
+    elif result == 2:
+        pass #create message that username already exists
+    else:
+        pass #username and password too short or long
+
+def login_user(request):
+    result = 1
+    if request.method == 'POST':
+        username = request.POST['username']
+        password = request.POST['password']
+        result = login(username, password)
+    if result == 0:
+        global current_user
```

Comment:
I am a little concerned about having a global user here. What happens when two people are logged in? Will one of them overwrite the other? The frontend will pass the user information whenever they make a request to the backend, so I do not think the backend will need to save the user.

Reply:
Ok, I think you're right about this.